core/src/main/java/com/scalar/db/config/DatabaseConfig.java

```diff
@@ -54,6 +54,7 @@ public class DatabaseConfig {
   public static final String CROSS_PARTITION_SCAN = SCAN_PREFIX + "enabled";
   public static final String CROSS_PARTITION_SCAN_FILTERING = SCAN_PREFIX + "filtering.enabled";
   public static final String CROSS_PARTITION_SCAN_ORDERING = SCAN_PREFIX + "ordering.enabled";
+  public static final String DEFAULT_SYSTEM_NAMESPACE_NAME = "scalardb";
```

Comment:
Now that `CassandraAdmin` is using such a constant, all storage does. So I removed a similar constant defined in each storage Admin to use this global one instead.